VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -531,7 +537,7 @@ def do_status_report(self, operation, status, status_code, message):
                 if "VMRestartPending" in encryption_status:
                     stat[0]["status"]["formattedMessage"]["message"] = "OS disk successfully encrypted, please reboot the VM"
                     
-            stat_rept = json.dumps(stat)
+            stat_rept = json.dumps(stat, ensure_ascii=False)
```

Comment:
investigate method signature, add comment explaining ensure_ascii=false
or just make var ascii before sending

Reply:
# use default encode_ascii of true, then decode for python2 + python3 compat
stat_rept = json.dumps(stat).decode('utf-8')